### ISSUE TITLE
Not getting localized strings on iOS 13

### ISSUE THREAD

**Issue by ZaLiTHkA** [kelvinhokk/cordova-plugin-localization-strings#55]
Greetings.. I'm having some difficulty getting this plugin to play nicely on an iPad running iOS 13.2.3. For what it's worth, it *is* working exactly as expected on my Xiaomi phone running Android 9 though. The project itself is generated with Ionic 4, using Angular 8 for the UI side and Cordova 9 for native platform integration.

First off, just some context: I do not have access to a new enough Mac to build locally for iOS 13 (I've got a really old Mac Mini, but I can't upgrade iOS to run the latest Xcode release), so I'm building the `ipa` files in the cloud using Ionic Package.

To summarise, the app display name and usage descriptions are still those defined in my `package.json`, the development build does not use the strings defined in the localization files.

---

I added the plugin using `ionic cordova plugin cordova-plugin-localization-strings`, created the expected `/translations/app/[locale].json` files for English and German, and configured each one with only the following:

```json
{
  "config_ios": {
    "NSBluetoothPeripheralUsageDescription": "Use Bluetooth description in English",
    "NSPhotoLibraryUsageDescription": "Read Photo Library description in English",
    "CFBundleDisplayName": "Name in English",
    "CFBundleName": "Name in English"
  },
  "config_android": {
    "app_name": "Name in English"
  }
}
```

Initially I had actually removed the `BLUETOOTH_USAGE_DESCRIPTION` and `PHOTO_LIBRARY_USAGE_DESCRIPTION` keys from my `package.json`, thinking that it would be "safer" to not have the build process set one string only to have it replaced afterwards. Then for testing, I re-added these permission descriptions to `package.json` with an extra note at the end to differentiate the sources, and running it on the iPad definitely doesn't use the strings defined in my localization files.

I went searching for information on Cordova Hooks support in Ionic Package (which is part of the AppFlow side of Ionic Pro), from which I discovered a plugin called [`ionic-package-hooks`](https://github.com/ionic-team/ionic-package-hooks), but this was deprecated July last year with a message stating that "Ionic Pro lets you use any Cordova hooks you want".

Just for the sake of clarity, I don't get any errors in the build output, it appears to add the plugin and run the hooks as expected:

```
[06:29:53]: ▸ Discovered saved plugin "cordova-plugin-localization-strings". Adding it to the project
[06:29:54]: ▸ Installing "cordova-plugin-localization-strings" for ios
[06:29:54]: ▸ Adding cordova-plugin-localization-strings to package.json
[06:29:54]: ▸ new pbx project written with localization groups
[06:29:54]: ▸ /Users/ionic/builds/project-0/platforms/ios purged from project cache
```

Also, please note that this plugin is at the end of the Cordova plugins listed, so it's processed after all the others, which (as I understand) means if this changes the same thing as any other plugin before it, this *should* overwrite whatever was there.

---

Does anybody have a suggestion on what I can try here...?

**Comment by ZaLiTHkA** [comment]
Note: I'm not sure if I'm having the same issue as that described in #49, simply because I can't get it to work in development, so I haven't tried a production build yet.. Ionic Package uses GitLab CI Runners for it's build pipeline, so it creates a new VM on each run, which means everything is installed fresh on every build.

**Comment by ZaLiTHkA** [comment]
🤦‍♂ so, the Android test I did was over the weekend at home with a new project. the SSH keys for access to our cloud build instance are not configured on my personal PC, so the iOS test was with a different project now.

turns out, when I implemented in no this morning, I put the localization files in `[project_root]/src/translations/app`, not `[project_root]/translations/app`...

I've just moved the folder to the correct place and rebuilt and it appears to be working on iOS now, for both the app name and usage descriptions.
